fs2/src/main/scala/laserdisc/fs2/RedisConnection.scala

```diff
@@ -4,14 +4,14 @@ package fs2
 import java.net.InetSocketAddress
 import java.nio.channels.AsynchronousChannelGroup
 
-import _root_.fs2.io.tcp
 import _root_.fs2._
+import _root_.fs2.io.tcp
 import cats.Applicative
 import cats.effect.Effect
 import cats.syntax.all._
+import laserdisc.protocol.RESP
```

Comment:
Missed this, not needed AFAICT
`RESP` type is forwarded from package object (see [here](https://github.com/laserdisc-io/laserdisc/blob/master/core/src/main/scala/laserdisc/package.scala#L20))